ui/components/multichain/app-header/app-header.js

```diff
@@ -64,12 +75,17 @@ export const AppHeader = ({ onClick }) => {
     getEnvironmentType() === ENVIRONMENT_TYPE_POPUP &&
     origin &&
     origin !== browser.runtime.id;
+  const showProductTour =
+    completedOnboarding && !onboardedInThisUISession && showProductTourPopup;
+  const productTourDirection = document
+    .querySelector('[dir]')
+    ?.getAttribute('dir');
```

Comment:
Do we know if MetaMask sets a `dir` attribute for RTL languages?  For LTR it doesn't appear we do.

Reply:
Yup, I believe the default language when someone installs MetaMask is English but once we update the language, MetaMask sets a `dir` attribute. 


Uploading Screen Recording 2023-04-21 at 12.48.31 AM.mov…

Reply:
Ahh, I was looking at the wrong node, nice!